frontend/packages/console-shared/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -17,6 +17,7 @@ import {
 
 type DropdownItemProps = {
   model: K8sKind;
+  resource: K8sResourceKind;
```

Comment:
should this be optional?

Reply:
resource is new field which is always passed from this component https://github.com/openshift/console/blob/977aca12752a297cac1124d0d106313ff569ab4b/frontend/packages/console-shared/src/components/dropdown/ResourceDropdown.tsx#L174.